tests/integration/specs/visual-regression/visual-regression.spec.js

```diff
@@ -66,6 +76,9 @@ function visitStory(uuid, story, controls) {
     cy.get(waitOn);
   }
 
+  // Wait for stories to be fully loaded
+  cy.wait(800);
```

Comment:
This will make the tests flakey on slow machines or slow on powerful machines. We should wait on a specific elements to be rendered and continue immediately, e.g. I see that each story is rendered inside a div with id `#storybook-root`. Could we use that? Can we use 

```js
if (waitOn) {
    cy.get(waitOn);
  } else {
  cy.get('#storybook-root');
}
```

Cypress `get` function tries to find element in the page and retries several times before giving up and throwing an error. The `defaultCommandTimeout` is set to 3s for Cypress so it will give plenty of time to render but it will not waste the time waiting 800ms if the story is already rendered.

Reply:
We can control individual stories by passing custom waitOn selectors, e.g.

```
allIcons.parameters = {
  visualRegressionTests: {
    waitOn: '.search-results',
    snapshotElement: '.search-results',
    controls: {
      large: { size: 64 },
      'type-error': { type: IconType.ERROR },
    },
  },
};
```

You can wait on `.v-popper__popper` class or add a `data-test` to the `<h1>Test Popover<h1>` and await that one.

How many stories are affected? We shouldn't add this to every story.

Reply:
I can't tell the exact number of stories that are affected since it varies in each run and are not always the same stories. I will run the test a couple of time using this approach to see if it works and how many stories need it